src/microbe_stage/Microbe.cs

```diff
@@ -1352,7 +1352,9 @@ public override void _IntegrateForces(PhysicsDirectBodyState physicsState)
     {
         // TODO: should movement also be applied here?
 
-        physicsState.Transform = GetNewPhysicsRotation(physicsState.Transform);
+        // physicsState.Transform = GetNewPhysicsRotation(physicsState.Transform);
```

Comment:
If this isn't code to be definitely remove, it would be suited to tell why it is just commented.